This patch has been integrated into Liberty but has not yet been
back-ported to Kilo.

commit c8b597f91250300fe28ce55990638f81b316e77d
Author: Masco Kaliyamoorthy <masco.kaliyamoorthy@enovance.com>
Date:   Tue Apr 7 16:06:18 2015 +0530

    hide disassociate floating ip when no ip attached

    hide the disassociate floating ip action,
    when the instance is not attached with any floating ip.

    Change-Id: Ifdb8aefcbc1791614c5d943e5081ac1efdd7acd6
    Closes-Bug: #1441088

--- horizon-2015.1.2/openstack_dashboard/dashboards/project/instances/tables.py.orig	2016-04-12 08:16:23.384881401 -0700
+++ horizon-2015.1.2/openstack_dashboard/dashboards/project/instances/tables.py	2016-04-12 08:16:42.245770950 -0700
@@ -620,7 +620,11 @@ class SimpleDisassociateIP(policy.Policy
             return False
         if not conf.HORIZON_CONFIG["simple_ip_management"]:
             return False
-        return not is_deleting(instance)
+        for addresses in instance.addresses.values():
+            for address in addresses:
+                if address.get('OS-EXT-IPS:type') == "floating":
+                    return not is_deleting(instance)
+        return False
 
     def single(self, table, request, instance_id):
         try:
